cmd/ovsdb-mon/shell.go

```diff
@@ -268,12 +274,29 @@ func (s *OvsdbShell) Run(ovsPtr *client.Client, args ...string) {
 					panic(err)
 				}
 			},
-			Completer: func(args []string) []string {
-				return tableFields[tableName]
+			CompleterWithPrefix: func(prefix string, args []string) []string {
+				if prefix == "" {
+					return exactTableFields[tableName]
+				}
+				return allTableFields[tableName]
 			},
 		}
 		listCmd.AddCmd(&subTableCmd)
 	}
+
+	// The list command autocompleter returns the exact table names if user has not
+	// started typing. If she has, then also include aliases (which are lower cased versions
```

Comment:
NIT: she/he  (be nice to the dudes too!) :)

Reply:
OK. I'll put both. I tend to default to "she" for inclusiveness (dudes are not underrepresented ;))